specs/0116-LLM-assisted-translations.md

```diff
@@ -14,23 +14,24 @@ Modern Large Language Models (LLMs) — like OpenAI’s GPT-4 — have reached a
 
 In the Machinery tab of the Pontoon translate view, users will encounter a new drop-down option for each machine translation suggestion for a given string, initially offering three choices: 
 
-1) `MAKE INFORMAL` - Provide a more informal version of this translation 
+1) `REPHRASE` - Provide an alternative to this translation 
 2) `MAKE FORMAL` - Provide a more formal version of this translation
-3) `REPHRASE` - Provide an alternative to this translation 
+3) `MAKE INFORMAL` - Provide a more informal version of this translation  
 
 Upon selecting any of these options, the original machine translation will temporarily show `AI is writing` as the LLM generates the new translation. This revised translation will then appear in place of the original suggestion. Once a new translation is generated, another option `SHOW ORIGINAL` will be seen in the drop-down menu. Users can revert to the original translation by selecting the `SHOW ORIGINAL` option from the drop-down menu. 
 
 Additionally, after selecting an option from the drop-down menu, the label beside the drop-down arrow will change as follows:
-- `MAKE INFORMAL` -> `INFORMAL`
-- `MAKE FORMAL` -> `FORMAL`
 - `REPHRASE` -> `REPHRASED`
+- `MAKE FORMAL` -> `FORMAL`
+- `MAKE INFORMAL` -> `INFORMAL`
+
 - `SHOW ORIGINAL` -> Reverts to the initial drop-down state
 
 # Implementation notes
 
 The translation enhancement options will utilize OpenAI’s GPT-4 API to improve the existing machine translation suggestions. This is especially useful for accurately translating challenging phrases, like idiomatic expressions, by utilizing the LLM's capability to adapt translations to the specific style and nuances of the target language.
 
-Additionally, data on the frequency of LLM usage and the adoption rate of LLM-provided translations will be collected to assess the tool's impact and refine its capabilities. This will include identifying additional machinery sources to better understand which aspects of the tool are most utilized. Furthermore, we will log user experience (UX) actions for each selection made in the drop-down menu.
+Additionally, data on the frequency of LLM usage and the adoption rate of LLM-provided translations will be collected to assess the tool's impact and refine its capabilities. This will include identifying additional Machinery sources, such as "LLM", to better understand if the feature is being utilized. Furthermore, we will log user experience (UX) actions for each selection made in the drop-down menu.
```

Comment:
Is `storing` better than `identifying`, given your explanation?

Reply:
Yes, that makes more sense.